src/objects/list/list.stories.mdx

```diff
@@ -0,0 +1,60 @@
+import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
+import { withKnobs, array, select, text } from '@storybook/addon-knobs';
+import template from './list.twig';
+const defaultItems = [
+  'Design Tokens',
+  'Objects',
+  'Components',
+  'Scope',
+  'Utilities',
+  'Themes',
+];
+const tagNames = ['ul', 'ol'];
+
+<Meta title="Objects/List" decorators={[withKnobs]} />
+
+# List
+
+The `o-list` object presents list items in a straightforward and unopinionated manner. While it can be applied to any container, it was designed specifically for `<ul>` and `<ol>` elements.
+
+**Important:** Remember to include the `role="list"` attribute on `<ul>` elements using this pattern to avoid [an issue with unstyled lists and Safari/VoiceOver](https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html).
```

Comment:
Once again, excellent comment that answered a question I was just about to type up!